src/containers/Experiment/ProcessingInformationCell.js

```diff
@@ -4,319 +4,259 @@ import ModalManager from '../../components/Modal/ModalManager';
+                <td>{version}</td>
+              </tr>
+            ))}
+
+            <GenomeBuild organism={sample.organism.name} />
+          </tbody>
+        </table>
+
+        <section className="processing-info-modal__section">
+          <SubmitterSuppliedProtocol {...this.props} />
+        </section>
+      </div>
+    );
+  }
+
+  static PROTOCOLS = {
+    'Affymetrix SCAN': AffymetrixScanProtocol,
+    'Illumina SCAN': IlluminaScanProtocol,
+    'Transcriptome Index': null,
+    'Salmon Quant': SalmonProtocol,
+    MultiQC: null,
+    Tximport: TxtimportProtocol,
+
+    'Submitter-processed': SubmitterSuppliedProtocol
+  };
+
+  _getProtocolDescription(name) {
+    const Component = ProcessingInformationModalContent.PROTOCOLS[name];
+    if (!Component) {
+      return null;
+    }
+    return <Component {...this.props} />;
+  }
 }
 
-function SubmitterSuppliedProtocol() {
+function GenomeBuild({ organism }) {
   return (
-    <div>
-      <h3>Submitter Supplied Protocol</h3>
-      <p>
-        These tissues samples were obtained at surgery and stored at -80C until
-        use., These tissues samples were obtained at surgery without any other
-        pretreatment., Acid guanidinium thiocyanate-phenol-chloroform extraction
-        of total RNA was performed according to the previous report (Anal.
-        Biochem, 162: 156,1987)., Biotinylated cRNA were prepared according to
-        the standard Affymetrix protocol from 2 ug total RNA (Expression
-        Analysis Technical Manual, 2001, Affymetrix)., Title: Affymetrix CEL
-        analysis. Description:, The data were analyzed with Microarray Suite
-        version 5.0 (MAS 5.0) using Affymetrix default analysis settings and
-        global scaling as normalization method.
-      </p>
-    </div>
+    <Loader fetch={() => getGenomeBuild(organism)}>
+      {({ isLoading, data }) =>
+        !isLoading && !data ? (
+          <React.Fragment />
+        ) : (
+          <tr>
+            <td className="processing-info-modal__version">Genome build</td>
+            <td>{data || '...'}</td>
+          </tr>
+        )
+      }
+    </Loader>
   );
 }
 
-function GeneIdentifierConversion() {
+function AffymetrixScanProtocol() {
   return (
-    <div>
-      <h3>Gene Identifier Conversion</h3>
+    <div className="processing-info-modal__protocol-description">
+      <h3>SCAN</h3>
       <p>
-        The gene identifiers were detected and converted to Ensembl gene IDs
-        using our{' '}
+        SCAN (Single Channel Array Normalization) is a normalization method for
+        single channel (Affymetrix) microarrays that allows us to process
+        individual samples. SCAN models and corrects for the effect of technical
+        bias, such as GC content, using a mixture-modeling approach. For more
+        information about this approach, see the primary publication (Piccolo,
+        et al. Genomics. 2012.
```

Comment:
Not sure whether we need to add more formats to this paragraph (such as use italic font for `Genomics. 2012`). Any thoughts, @jaclyn-taroni?

Reply:
+1 _Genomics._ 2012.

We'll probably want to be consistent in italicizing journal names in all protocols, then.